data/templates/autounattend_windows_2012_r2_SMC_6028U-TR4+.xml

```diff
@@ -84,7 +84,7 @@
             </AutoLogon>
             <FirstLogonCommands>
 	       <SynchronousCommand wcm:action="add">
-                    <CommandLine>powershell -Command "Invoke-WebRequest -uri http://<%=server%>:<%=port%>/api/current/templates/renasar-ansible.pub"</CommandLine>
+                    <CommandLine>powershell -Command "curl -Method POST -ContentType 'application/json' http://<%=server%>:<%=port%>/api/current/notification?nodeId=<%=nodeId%> -Outfile windows-install.log"</CommandLine>

```

Comment:
may consider replace -Method with a simple -X to keep consistent with other places.